bound/kt/src/main/kotlin/tbdex/sdk/web5/PresentationDefinition.kt

```diff
@@ -1,40 +1,54 @@
 package tbdex.sdk.web5
 
-import tbdex.sdk.rust.InputDescriptorData as RustCoreInputDescriptor
+import com.fasterxml.jackson.annotation.JsonProperty
+import tbdex.sdk.Json
 import tbdex.sdk.rust.PresentationDefinition as RustCorePresentationDefinition
-import tbdex.sdk.rust.PresentationDefinitionData as RustCorePresentationDefinitionData
 
-typealias InputDescriptor = RustCoreInputDescriptor
-
-class PresentationDefinition {
-    val id: String
-    val name: String?
-    val purpose: String?
+data class PresentationDefinition(
+    val id: String,
+    val name: String?,
+    val purpose: String?,
+    @JsonProperty("input_descriptors")
     val inputDescriptors: List<InputDescriptor>
-
-    val rustCorePresentationDefinition: RustCorePresentationDefinition
-
-    constructor(id: String, name: String? = null, purpose: String? = null, inputDescriptors: List<InputDescriptor>) {
-        this.id = id
-        this.name = name
-        this.purpose = purpose
-        this.inputDescriptors = inputDescriptors
-
-        this.rustCorePresentationDefinition = RustCorePresentationDefinition(
-            RustCorePresentationDefinitionData(id, name, purpose, inputDescriptors)
-        )
-    }
-
-    constructor(rustCorePresentationDefinitionData: RustCorePresentationDefinitionData) {
-        this.id = rustCorePresentationDefinitionData.id
-        this.name = rustCorePresentationDefinitionData.name
-        this.purpose = rustCorePresentationDefinitionData.purpose
-        this.inputDescriptors = rustCorePresentationDefinitionData.inputDescriptors
-
-        this.rustCorePresentationDefinition = RustCorePresentationDefinition(rustCorePresentationDefinitionData)
-    }
+) {
+    internal val rustCorePresentationDefinition = RustCorePresentationDefinition(
+        Json.stringify(this)
+    )
 
     fun selectCredentials(vcJwts: List<String>): List<String> {
         return this.rustCorePresentationDefinition.selectCredentials(vcJwts)
     }
-}
\ No newline at end of file
+}
+
+data class InputDescriptor(
```

Comment:
This is in mainline web5-rs now, so idk when the time will come when we take this out and use web5-rs

Reply:
Yeah we need to make that jump sooner rather than later, but I'm not prepared to full-commit to a hosting solution. @nitro-neal what do you think about a temporary measure like throwing something up to jitpack, with recognition it isn't our longterm hosting solution?